python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/router_isis.schema.yml

```
@@ -44,6 +44,69 @@ keys:
                   - str
                 default: 10000
                 description: Delay in milliseconds.
+          lsp:
+            description: Link State Packet timers.
+            type: dict
+            keys:
+              generation:
+                type: dict
+                keys:
+                  interval:
+                    type: int
+                    convert_types:
+                      - str
+                    min: 1
+                    max: 300
+                    required: true
+                    description: Maximum interval (in seconds) between generating two LSPs.
+                  initial_wait_time:
+                    type: int
+                    convert_types:
+                      - str
+                    min: 1
+                    max: 300000
+                    description: Initial wait time (in milliseconds) before generating LSPs.
+                  wait_time:
+                    type: int
+                    convert_types:
+                      - str
+                    min: 1
+                    max: 300000
+                    description: Wait time (in milliseconds) between generating the first and second LSPs.
+              out_delay:
+                type: int
+                convert_types:
+                  - str
+                min: 1
+                max: 65000
+                description: Transmit delay (in milliseconds) for link state packets.
+              refresh_interval:
+                type: int
+                convert_types:
+                  - str
+                min: 30
+                max: 65535
+                description: Interval (in seconds) between two LSP refreshes.
+              min_remaining_lifetime:
+                type: int
+                convert_types:
+                  - str
+                min: 60
+                max: 65535
+                description: Minimum remaining lifetime for LSPs (in seconds).
+          csnp_generation:
+            type: dict
+            keys:
+              interval:
+                type: int
+                convert_types:
+                  - str
+                min: 1
+                max: 300
+                description: Transmit frequency (in seconds) for CSN packets.
+              p2p_disabled:
+                type: bool
+                description: Disable periodic CSN packets for P2P links.
```

I discussed this with other maintainers on a calll ; from a consistency point of view with `lsp` implemented in the same PR let's use a "more than necessary" nested dictionary here.

```suggestion
          csnp:
            type: dict
            keys:
              generation:
                type: dict
                keys:
                  interval:
                    type: int
                    convert_types:
                      - str
                    min: 1
                    max: 300
                    description: Transmit frequency (in seconds) for CSN packets.
                  p2p_disabled:
                    type: bool
                    description: Disable periodic CSN packets for P2P links.
```